catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceDownloadManager.java

```diff
@@ -150,6 +150,7 @@ public void setMaxRetryAttempts(int maxRetryAttempts) {
     }
 
     public void setDelayBetweenAttempts(int delayBetweenAttempts) {
+        LOGGER.debug("We called the delay between attempts: {}", delayBetweenAttempts);
```

Comment:
Can we change the message to `Delay between attempts set to {}` or something like that? `We`, `I` and `You` in log messages always feels weird.